src/components/ThankYouResponse.tsx

```diff
@@ -0,0 +1,16 @@
+import { h } from 'preact';
+
+type Props = {
+  title: string;
+  subtext: string;
+};
+
+export const ThankYouResponse = ({ title, subtext }: Props) => {
+  return (
+    <div className={'text-center px-8 min-w-[489px]'}>
+      <p className={'text-5xl'}>❤️</p>
```

Comment:
This should be a `svg` icon, since unicode representations vary for each browser.